app/js/components/identifiers/identifiers.jsx

```diff
@@ -249,14 +249,23 @@ export default class Identifiers extends React.Component {
             return (
               <div key={id.uuid}>
                 {!this.state.hideViewCard &&
-                  <div className="viewCard">
-                    {id.identifierType.display}{': '}
-                    {this.state.activeCard !== index ? id.identifier : editIdentifiers.identifier || id.identifier}
-                    {'   '}
-                    {this.state.editState === false &&
-                      <a onClick={() => this.callEdit(index, id.uuid)}>Edit
-											</a>
+                  <div>
+                    {
+                      id.preferred &&
+                      <div className="preffered">
+                        <span className="badge badge-info">Preferred</span>
+                      </div>
                     }
+
+                    <div className="viewCard">
+                      {id.identifierType.display}{': '}
+                      {this.state.activeCard !== index ? id.identifier : editIdentifiers.identifier || id.identifier}
+                      {'   '}
+                      {this.state.editState === false &&
+                        <a onClick={() => this.callEdit(index, id.uuid)}>Edit
+											</a>
+                      }
+                    </div>
```

Comment:
The closing anchor tag seems to be indented too much

Reply:
corrected